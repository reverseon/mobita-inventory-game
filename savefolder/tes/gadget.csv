id;nama;deskripsi;jumlah;rarity;tahun_ditemukan
G1;Simp Wand;Simping Alternative for Witch;20;B;2004
G2;Simping Lightstick;For HoneyWorks Concert;20;A;2001
G3;Ena Komiya;Istri Admin;1;S;2021
G4;Ai Hayasaka;Istri Admin Juga;1;S;2021
G20;Amatsuka Uto;Istri Admin;1;S;2020
G5;Mary Saotome;yes;1;S;2020
G6;Yumeko Jabami;yes;1;S;2020
G7;Gabriel;GBO;1;S;2020
