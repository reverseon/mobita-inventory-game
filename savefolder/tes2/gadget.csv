id;nama;deskripsi;jumlah;rarity;tahun_ditemukan
G1;Simp Wand;Simping Alternative for Witch;20;B;2004
G2;Simping Lightstick;For HoneyWorks Concert;20;A;2001
G3;Ena Komiya;Istri Admin;1;S;2021 
G4;Ai Hayasaka;Istri Admin Juga;1;S;2021
