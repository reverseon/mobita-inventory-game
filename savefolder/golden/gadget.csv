id;nama;deskripsi;jumlah;rarity;tahun_ditemukan
G3;Ena Komiya;Istri Admin;1;S;2021
G4;Ai Hayasaka;Istri Admin Juga;1;S;2021
G5;Mary Saotome;yes;60;S;2020
G6;Yumeko Jabami;yes;1;S;2020
G7;Gabriel;GBO;21;S;2020
G8;Charlotte;Yes;70;A;2020
G9;Hina Amano;100% Sunshine Girl;138;B;2020
G12;Nao Tomori;Charlotte;8;S;2002
G10;Naruto;Kagebunshin;20;A;2010
G11;Bocchi;Lonely Bocchi;100;B;2001
